scipy/sparse/_construct.py

```diff
@@ -1244,15 +1244,16 @@ def data_sampler(size):
     return vals, ind
 
 
+@_transition_to_rng("random_state")
```

Comment:
Because `random_state` has been accepted by position, too:
```suggestion
@_transition_to_rng("random_state", position_num=5)
```
(I could have used introspection in the wrapper, but that seems to make people uncomfortable.)